ui-ngx/src/assets/locale/locale.constant-ca_ES.json

```diff
@@ -632,7 +632,7 @@
         "attributes": "Atributs",
         "latest-telemetry": "Última telemetria",
         "attributes-scope": "Abast dels atributs del dispositiu",
-        "scope-latest-telemetry": "Última telemetria",
```

Comment:
Please leave `scope-latest-telemetry` as is. It may be used somewhere in custom widgets or dashboards. Just add the one you need.